packages/yoga/src/Menu/web/MenuList.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { forwardRef } from 'react';
+import styled from 'styled-components';
+import { Content as MenuListRoot } from '@radix-ui/react-dropdown-menu';
+import { node, string } from 'prop-types';
+
+import Box from '../../Box';
+
+const StyledMenuList = styled(MenuListRoot)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  ${({
+    theme: {
+      yoga: {
+        components: { menu },
+      },
+    },
+  }) => {
+    return `
+    margin: ${menu.margin.default}px 0;
+    padding: 0;
+    min-width: ${menu.width.min}px;
+    max-width: ${menu.width.max}px;
+    border-radius: ${menu.border.radius}px;
+
+    background-color: ${menu.backgroundColor.white};
+    box-shadow: 0px 2px 6px rgba(152, 152, 166, 0.25);
```

Comment:
Great contribution @guilhermecardoso-gympass ! 🚀 

here should we use the [elevation token](https://gympass.github.io/yoga/guidelines/tokens/elevations)?

Reply:
this shadow not have in elevation token! at least I didn't find

Reply:
caio is right about that gui, we should use the small elevation

Reply:
Fixed with a small elevation